lib/linter-js-yaml.js

```diff
@@ -0,0 +1,72 @@
+'use babel';
+
+import path from 'path';
+import { CompositeDisposable } from 'atom';
+import yaml from 'js-yaml';
+
+function processMessage(type, filePath, message) {
+  const line = message.mark.line;
+  const column = message.mark.column;
+  return {
+    type: type,
+    text: message.reason,
+    filePath: filePath,
+    range: [[line, column], [line, column + 1]],
+  };
+}
+
+class LinterJsYaml {
+  config = {
+    customTags: {
+      type: 'array',
+      default: [],
+      items: {
+        type: 'string',
+      },
+      description: 'List of YAML custom tags.',
+    },
+  }
+  activate() {
+    require('atom-package-deps').install('linter-js-yaml');
+    this.subscriptions = new CompositeDisposable();
+    this.subscriptions.add(atom.config.observe('linter-js-yaml.customTags', (customTags) => {
+      this.Schema = yaml.Schema.create(customTags.map((tag) => {
```

Comment:
In ES6 arrow functions, if you have only one parameter you can drop the parenthesis, `(tag) => ` can also be written as `tag => `